src/components/shared/painting-info-template/painting-info-template.jsx

```diff
@@ -138,9 +151,17 @@ export default function PaintingInfoTemplate({
         <RichText
           json={additionalDescription.json}
           classNames={styles.additionalDetails}
-          refs={additionalDescriptionRef}
```

Comment:
Removing this ref kills the existing functionality for paintings with additional description but doesn't have a video, does it not?

Reply:
The ref was only used for scrolling into view, but the scroll into view didn't account for the height of the nav bar so it was being slightly cut off. I moved the ref to a different location so when it scrolls into view the additional description wasn't cut off

Reply:
ohhh i see. Okay cool I like the thought, but that's a bit hacky, a ref should be attached to component it's meant for. If we were to move around components on this page, it would affect the behavior in an unexpected way. Can you either find a cleaner solution or put it back to the way it was?